book/requirements.txt

```diff
@@ -0,0 +1,13 @@
+geopandas==0.10.2
+matplotlib==3.5.1
+numpy==1.21.2
+pandas==1.3.5
```

Comment:
`pandas` v2 was released a year ago, so perhaps we should bump this? Are there other dependencies here that don't play nicely with v2?

Reply:
this file is not part of the github action